CHANGELOG.rst

```diff
@@ -344,7 +345,7 @@ All notable changes to this project are documented in this file.
 - various small bugfixes
 
 
-[0.4.3] 2017-12-21
+[0.4.4] 2017-12-21
```

Comment:
why change this old release number in the changelog?

Reply:
Because it was a duplicate from the previous release number and was throwing this warning:
`/Users/erik/Documents/code/neo-python/docs/source/changelog.rst:355: WARNING: duplicate label [0.4.3] 2017-12-21, other instance in /Users/erik/Documents/code/neo-python/docs/source/changelog.rst`

Reply:
I see. Hm, checking the file, the `0.4.3` section was just duplicated (in there 2x): https://github.com/CityOfZion/neo-python/blob/development/CHANGELOG.rst#043-2017-12-21

So the proper fix would be to just remove one of them :)

Reply:
Blast. Okay.